### PR TITLE
feat: notifications without account

### PR DESCRIPTION
**Pull request by shoom3301**

### CODE REVIEW

File: src/api/notification/content-types/notification/schema.json [hunk 1]
```diff
@@ -8,13 +8,13 @@
     "description": ""
   },
   "options": {
-    "draftAndPublish": true
+    "draftAndPublish": false
```
Comment (anxolin) [inline src/api/notification/content-types/notification/schema.json:11]: good call!

File: src/api/notification/services/notification.ts [hunk 2]
```diff
@@ -8,70 +8,83 @@ const MODULE_ID = 'api::notification.notification'
...
+}
+
 export default factories.createCoreService(MODULE_ID, ({ strapi }) => {
   return {
+    async getNotificationsForAll(push: boolean) {
+      return strapi.entityService.findMany(
+        MODULE_ID,
+        {
+          start: 0,
+          limit: 50,
```
Comment (anxolin) [inline src/api/notification/services/notification.ts:29]: what's the plan for the hardcoded 50

Comment (shoom3301) [reply src/api/notification/services/notification.ts:29]: I would even keep it as it is.
We could add pagination and so on, but...will we really need it?
I think the main use-case of the notifications is to notify users about something new which is relevant only for the nearest feature.
As a user I would read even only the last 3-5 notifications.
But this is only my assumption, I'm open for any proposal.

Comment (anxolin) [reply src/api/notification/services/notification.ts:29]: Okay, lets keep it for now them

**Pull request merged**